.github/workflows/lint.yml

```diff
@@ -16,10 +16,10 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
-      - name: Set up Go 1.19
+      - name: Set up Go
         uses: actions/setup-go@v3
         with:
-          go-version: 1.19
+          go-version: ^1.20
```

Comment:
Just to be clear, this is the same as `1.20.x` (any patch version) right?

Had to check the documentation:
* https://github.com/npm/node-semver#caret-ranges-123-025-004

Reply:
yes! same as in go.mod and the Dockerfile 👍